openrouteservice/pom.xml

```
@@ -44,11 +40,13 @@
         <url>https://github.com/GIScience/openrouteservice/issues</url>
     </issueManagement>
     <properties>
+        <java.version>11</java.version>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
```

instead of "hardcoding" the java-version again use the variable instead.
```suggestion
        <maven.compiler.source>${java.version}</maven.compiler.source>
        <maven.compiler.target>${java.version}</maven.compiler.target>
```